kurtosis_package/dockerfiles/contract_deployer.Dockerfile

```diff
@@ -1,46 +1,31 @@
-FROM debian:bookworm-slim AS foundry
+FROM debian:bookworm-slim
 
-# Install curl & git
-RUN apt update -y && \
-    apt upgrade -y && \
-    apt install -y curl git && \
-    apt clean
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
What's this for?

Reply:
https://github.com/Layr-Labs/avs-devnet/pull/221/commits/270b87405a7dd4b17119762fd33816e950127669